conf/application.conf

```diff
@@ -54,41 +54,16 @@ play.mailer {
 }
 
 # Security Filter Configuration - Content Security Policy
-# ~~~~~
-#play.filters.headers.contentSecurityPolicy
-# default-src
-# 'self'
-# img-src
-# 'self'
-# *.fbcdn.net (Facebook)
-# *.twimg.com (Twitter)
-# *.googleusercontent.com (Google)
-# *.xingassets.com (Xing)
-# vk.com (VK)
-# *.yimg.com (Yahoo)
-# secure.gravatar.com
-# style-src
-# 'self'
-# 'unsafe-inline'
-# cdnjs.cloudflare.com
-# maxcdn.bootstrapcdn.com
-# cdn.jsdelivr.net
-# fonts.googleapis.com
-# 'unsafe-inline' (in-line css found in bootstrap.min.js)
-# font-src
-# 'self'
-# fonts.gstatic.com
-# fonts.googleapis.com
-# cdnjs.cloudflare.com
-# script-src
-# 'self'
-# clef.io
-# connect-src
-# 'self'
-# twitter.com
-# *.xing.com
-# frame-src
-# clef.io
-play.filters.headers.contentSecurityPolicy="default-src 'self'; img-src 'self' data: *.fbcdn.net *.twimg.com *.googleusercontent.com *.xingassets.com vk.com *.yimg.com secure.gravatar.com; style-src 'self' cdnjs.cloudflare.com maxcdn.bootstrapcdn.com cdn.jsdelivr.net fonts.googleapis.com; font-src 'self' fonts.gstatic.com fonts.googleapis.com cdnjs.cloudflare.com; script-src 'self' clef.io cdnjs.cloudflare.com; connect-src 'self' twitter.com *.xing.com; frame-src clef.io"
+play {
+  filters.headers {
+  contentSecurityPolicy = "default-src 'self';"
```

Comment:
The following lines have a wrong indentation. And I think the first key should be play.filters.headers because in the configuration there are other keys starting with play but they are not grouped together.